setup.py

```diff
@@ -16,9 +16,7 @@
         'cpg-utils>=5.1.1',
         'cyvcf2==0.30.18',
         'analysis-runner>=2.43.3',
-        # Pin Hail to the release prior to 0.2.133
-        # see https://centrepopgen.slack.com/archives/C018KFBCR1C/p1731047651300539
-        'hail==0.2.132',
+        'hail>=0.2.133',
```

Comment:
Best to put the previous comment back too:

```suggestion
        'hail>=0.2.133',  # Pin Hail at CPG's installed version
```

Traditionally we use `==` for this particular dependency. I don't know that it matters too much, but perhaps we should stick with tradition…